app/views/meals/_meal.html.erb

```diff
@@ -0,0 +1,29 @@
+<tr class=<%= 'table-success' if meal.active %>>
+  <td><%= meal.id %></td>
+  <td><%= image_tag meal.image, height: 50 if meal.image.attached? %></td>
+  <td><%= meal.name %></td>
+  <td><%= meal.price %></td>
+  <td><%= meal.non_veg ? "Non-Veg" : "Veg" %></td>
+  <td><%= truncate(meal.description, length: 30)%></td>
+  <td>
+    <div class= "d-flex">
+      <% if meal.active %>
+        <%= link_to toggle_inactive_meal_path(meal), class: 'btn btn-success btn-sm me-2', role: 'button' do %>
```

Comment:
Also can we add tooltip saying activate/deactivate on the icon?

Reply:
Done

Reply:
@rishabh-vinsol 
tooltip not working